packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/raw/BaseLegacyRawManifest.kt

```diff
@@ -37,4 +37,40 @@ abstract class BaseLegacyRawManifest(json: JSONObject) : RawManifest(json) {
 
   @Throws(JSONException::class)
   override fun getSDKVersion(): String = json.getString("sdkVersion")
+
+  override fun getExpoGoConfigRootObject(): JSONObject? {
+    return json
+  }
+
+  override fun getExpoClientConfigRootObject(): JSONObject? {
+    return json
+  }
+
+  override fun getSlug(): String? = if (json.has("slug")) {
+    json.optString("slug")
+  } else {
+    null
+  }
+
+  override fun getAppKey(): String? = if (json.has("appKey")) {
+    json.optString("appKey")
+  } else {
+    null
+  }
+
+  fun getCommitTime(): String? = if (json.has("commitTime")) {
+    json.optString("commitTime")
+  } else {
+    null
+  }
+
+  @Throws(JSONException::class)
+  private fun getPublishedTime(): String = json.getString("publishedTime")
+
+  override fun getSortTime(): String? {
+    // use commitTime instead of publishedTime as it is more accurate;
+    // however, fall back to publishedTime in case older cached manifests do not contain
+    // the commitTime key (we have not always served it)
+    return getCommitTime() ?: getPublishedTime()
```

Comment:
we should be able to safely remove this fallback, as we've been serving both fields for the entirety of the lifetimes of all SDK versions we currently support in Expo Go. I don't think the expo-updates runtime even looks at `publishedTime`.

Reply:
Will follow-up: https://linear.app/expo/issue/ENG-1526/remove-published-time-fallback-in-manifest-code